src/components/content/courses/DraftCourseDetails.js

```
@@ -9,407 +9,467 @@ import axios from 'axios';
+    const event = new NDKEvent(ndk);
+    event.kind = 30004;
+    event.content = '';
+    event.tags = [
+      ['d', courseId],
+      ['name', title],
+      ['picture', coverImage],
+      ['image', coverImage],
+      ['description', summary],
+      ['l', 'course'],
+      ['price', price.toString()],
+      ...processedEvent?.topics?.map(topic => ['t', topic]),
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)